docker-compose.prod-run.yml

```diff
@@ -3,12 +3,14 @@ version: "3.7"
 services:
   frontend:
     image: etimodanielwinther/diamonds2_frontend:${DIAMONDS_DOCKER_TAG}
+    restart: always
```

Comment:
Dont know if this restarts container when docker is restarted. But please try!

Reply:
Will do :)